<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mylink</class>
 <widget class="QWidget" name="mylink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>109</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>178</width>
    <height>109</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>178</width>
    <height>109</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_ip">
       <property name="text">
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ip"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_port">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_port">
       <property name="text">
        <string>7000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_link">
       <property name="text">
        <string>链接</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
